src/main/java/uk/gov/digital/ho/hocs/casework/api/CaseNoteService.java

```diff
@@ -58,4 +58,34 @@ public CaseNote createCaseNote(UUID caseUUID, String caseNoteType, String text)
         auditClient.createCaseNoteAudit(caseNote);
         return caseNote;
     }
+
+    public CaseNote updateCaseNote(UUID caseNoteUUID, String caseNoteType, String text) {
+        log.debug("Updating CaseNote: {}", caseNoteUUID);
+        CaseNote caseNote = caseNoteRepository.findByUuid(caseNoteUUID);
+        if (caseNote != null){
+            String prevCaseNoteType = caseNote.getCaseNoteType();
+            String prevText = caseNote.getText();
+            caseNote.setCaseNoteType(caseNoteType);
```

Comment:
We need to keep the lastUpdatedBy information + lastUpdated timestamp. This should then be fetch so that frontend can use this info.

Reply:
Fixed.